core/domain/suggestion_services.py

```diff
@@ -362,6 +363,9 @@ def accept_suggestion(
                 suggestion.suggestion_id)
         )
 
+    if suggestion.edited_by_reviewer:
+        commit_message = commit_message + ' (with edits)'
```

Comment:
Could you make this: `commit_message = '%s (with edits)' % commit_message`? We typically prefer using interpolation.

Reply:
Done